src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { Controller } from '@hotwired/stimulus'
-import { throttle } from 'lodash'
+import { throttle } from './utils'
 
```

Comment:
I think it would be even simpler to have the trigger declared inside the file (again, in the case of an import map, it's best not to expand unnecessarily across several files).

Reply:
I got it, but I also don't want to architecture the project just for import-maps and I guess it's completely fine to load this really small file

Removing lodash will make a huge difference 👍